### PR TITLE
feat: restore builds route

### PR DESCRIPTION
**Pull request by jithin1987**
## Context

Adds route `/builds/buildId` to UI which will redirect to actual url for the build. This is useful when debugging build errors when all you have is the build Id.

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const Router = EmberRouter.extend({
 /* eslint-disable array-callback-return */
 Router.map(function route() {
   this.route('home', { path: '/' });
+  this.route('builds', { path: '/builds/:build_id' });
```
Comment (tkyi) [inline app/router.js]: should this be `this.route('build', ` for consistency/since it takes you to one build?

Comment (jithin1987) [reply app/router.js]: that's inconsistent with rest api urls, ideally it should match. `/builds/1` takes you to a single build but it's not `build/1`

Comment (tkyi) [reply app/router.js]: Ah so it should be `this.route('pipelines',` for line 14 and `this.route('builds',` for line 16

Comment (jithin1987) [reply app/router.js]: yep, that's my 2 cents on this.

Comment (tkyi) [reply app/router.js]: Are we going to fix it?

Comment (jithin1987) [reply app/router.js]: later, that needs directory renaming and more unnecessary changes at this point.

**Pull request merged**